### ISSUE TITLE
Plugin doesn't work inside official terraform image (alpine)

### ISSUE THREAD

**Issue by Floby** [mrparkers/terraform-provider-keycloak#127]
When trying to run terraform with the keycloak-provider configured inside a docker container, the executable cannot be run.

It appears the released binary of the keycloak-provider is links to the glibc. However, the official terraform image is based on Alpine Linux and therefore uses musclibc.

We made it work that way with previous releases so I'm confident it's only a matter of rebuilding the latest version.

The complete behaviour within the container is :
```
> /provision # terraform init

Initializing the backend...

Initializing provider plugins...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
> /provision # terraform plan

Error: Failed to instantiate provider "keycloak" to obtain schema: fork/exec /root/.terraform.d/plugins/terraform-provider-keycloak_v1.8.0: no such file or directory

> /provision # ls -al /root/.terraform.d/plugins/terraform-provider-keycloak_v1.8.0 
-rwxr-xr-x    1 root     root      27065137 Jun 12 09:03 /root/.terraform.d/plugins/terraform-provider-keycloak_v1.8.0
> /provision # ldd /root/.terraform.d/plugins/terraform-provider-keycloak_v1.8.0
	/lib64/ld-linux-x86-64.so.2 (0x7fb0bab6a000)
	libpthread.so.0 => /lib64/ld-linux-x86-64.so.2 (0x7fb0bab6a000)
	libc.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7fb0bab6a000)

> /provision # /root/.terraform.d/plugins/terraform-provider-keycloak_v1.8.0
/bin/sh: /root/.terraform.d/plugins/terraform-provider-keycloak_v1.8.0: not found

> /provision # ldd /root/.terraform.d/plugins/terraform-provider-keycloak_v1.1.0
ldd: terraform-provider-keycloak_v1.1.0: Not a valid dynamic program


```

I'm not familiar enough with Go builds so I am unable to provide a fix for this :( but it used to work (last tested with version 1.1.0).

**Comment by Floby** [comment]
I installed `libc6-compat` as a workaround for the moment.

**Comment by mrparkers** [comment]
The 0.12 SDK was included with release 1.8.0 so it's possible that this is the reason.

I can add an extra release binary with CGO disabled which should work for alpine.  Could you test this by building with the environment variable `CGO_ENABLED=0`?
